migrations/20220818164120_create_anchor_and_request.js

```diff
@@ -0,0 +1,102 @@
+const REQUEST_TABLE_NAME = 'request'
+const TEMP_TABLE_NAME = 'temp'
+const ANCHOR_TABLE_NAME = 'anchor'
+
+const sortByStringKeys = (obj) =>
+  Object.fromEntries(Object.entries(obj).sort(([a], [b]) => a.localeCompare(b)))
+
+const checkObjEquality = (expected, received) =>
+  JSON.stringify(sortByStringKeys(expected)) === JSON.stringify(sortByStringKeys(received))
+
+const verifyTableStructure = async (knex, tableWithValidStructure, tableToCheck) => {
+  const [expectedStructure, existingStructure] = await Promise.all([
+    knex(tableWithValidStructure).columnInfo(),
```

Comment:
Instead of creating temp tables for comparison, you could just ship the column info dump as an object with the migration file.